roles/network/templates/network/wired-manage.conf

```
@@ -0,0 +1,3 @@
+# IIAB WiFi
+[keyfile]
+unmanaged-devices=interface-name:{{ item|trim }}
```

Lint for consistent style if possible?  (To match similar lines in NM-debian.yml and sysd-netd-debian.yml above.)

```suggestion
unmanaged-devices=interface-name:{{ item | trim }}
```